mathesar_ui/src/components/cell/cell-types/common/SteppedInputCell.svelte

```diff
@@ -113,7 +113,7 @@
     <div class="content">
       {#if value === null}
         <Null />
-      {:else if value}
+      {:else if value || value === 0}
```

Comment:
A better option would be to make sure that value is not undefined.

```suggestion
      {:else if value || typeof value !== 'undefined'}
```

This will remove the need of supporting `number` in text related cells. So we can remove the number type from `TextInput.svelte` and `TextBoxCell.svelte`. We have a dedicated issue for implementing a number cell, where we will be handling number inputs.

Reply:
Thank you @pavish, that's a good option as well. I have made necessary changes 👍